### PR TITLE
Register Gutenberg strings

### PR DESCRIPTION
**Pull request by brucepearson**
Register strings found in Gutenberg blocks.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5641

### CODE REVIEW

**Comment by brucepearson** [comment]
@andreasciamanna @adrianosferreira @grey-rsi @gordonsh24 Please review

File: src/class-wpml-gutenberg-integration.php [hunk 1]
```diff
@@ -2,6 +2,51 @@
...
+			}
+		}
+	}
+
+	private function get_string_id( $block ) {
+		if ( isset( $block['blockName'], $block['innerHTML'] ) && '' !== trim( $block['innerHTML'] ) ) {
+			return md5( $block['blockName'] . $block['innerHTML'] );
+		} else {
+			return null;
+		}
```
Comment (andreasciamanna) [inline src/class-wpml-gutenberg-integration.php:49]: @BruceOnTheGo can't you use this form?

```php
if ( isset( $block['blockName'], $block['innerHTML'] ) && '' !== trim( $block['innerHTML'] ) ) {
	return md5( $block['blockName'] . $block['innerHTML'] );
} 
return null;
```

The "else" is redundant.

**Review by andreasciamanna — APPROVED** [review]
I don't see the point of having always a if-else block when each block has a return point.

Other than that, I'm fine with this PR.

**Review by adrianosferreira — APPROVED** [review]

**Comment by brucepearson** [comment]
Merged to wpmltm-2112 branch

**Pull request closed without merging**